ebml_matroska.xml

```diff
@@ -63,7 +63,7 @@
       </element>
       TimecodeScale When combined with <a href="http://www.matroska.org/technical/specs/index.html#TimecodeScaleDenominator">TimecodeScaleDenominator</a> the Timestamp scale is given by the fraction TimecodeScale/TimecodeScaleDenominator in seconds.-->
       <element name="Duration" level="2" id="0x4489" type="float" minver="1" range="&gt; 0x0p+0">
-        <documentation lang="en">Duration of the Segment (based on TimecodeScale).</documentation>
+        <documentation lang="en" type="definition">Duration of the Segment measured in nanoseconds when scaled by TimecodeScale.</documentation>
```

Comment:
I find your new definition more confusing then the one it replaces. Basically the value contained in this element must be multiplied by `TimecodeScale`. The result is the duration in nanoseconds.

The specs use various terms to refer to such situations including but not limited to `based on TimecodeScale`, `scaled by TimecodeScale` and `scaled via TimecodeScale`, simply `scaled` and the corresponding negations. We should probably settle on one type of wording and add an explanatory section.

Maybe my confusion has more to do with my German background. In German we often use the word "wenn" for referring to both "when" and "if… then" (even though we also have "falls" which is much more appropriate for "if … then"). I guess my point is that I don't like introducing yet another choice of wording for scaling, and that we should really settle on a single choice of words.